src/py/mat3ra/made/tools/__init__.py

```diff
@@ -0,0 +1,61 @@
+from .bond_directions.bond_directions import BondDirections
+from .bond_directions.bond_directions_for_element_list import BondDirectionsForElementList
+from .bond_directions.bond_directions_templates_enum import BondDirectionsTemplatesEnum
+from .bond_directions.bond_directions_templates_for_element import BondDirectionsTemplatesForElement
+from .crystal_site import CrystalSite, CrystalSiteList
+from .modify import (
+    add_vacuum,
+    add_vacuum_sides,
+    filter_by_box,
+    filter_by_circle_projection,
+    filter_by_condition_on_coordinates,
+    filter_by_ids,
+    filter_by_label,
+    filter_by_layers,
+    filter_by_rectangle_projection,
+    filter_by_sphere,
+    filter_by_triangle_projection,
+    interface_displace_part,
+    interface_get_part,
+    remove_vacuum,
+    rotate,
+    translate_by_vector,
+    translate_to_center,
+    translate_to_z_level,
+    wrap_to_unit_cell,
+)
+from .optimize import evaluate_calculator_on_xy_grid
+from .third_party import (
```

Comment:
Why do we need to re-export third-party stuff here? It should be internal to made